### ISSUE TITLE
Show relative units as relative units

### ISSUE THREAD

**Issue by WileESpaghetti** [kdzwinel/CSS-Diff#11]
I like that CSS-Diff show computed differences, but what about when using %'s and ems?

It's great if I can see Div A is 100px wide and Div B is 202px wide, but it would be great if CSS-Diff would show me something that would let me know the reason A is 100px wide is because the stylesheet is setting it to width: 25% and that B is 202px wide because either I've set it to explicitly be 202px wide, or because I've set it to be width: 50.25%

**Comment by kdzwinel** [comment]
I agree, that would be great, but I'm working with `getComputedStyles` so I don't have this type of info. It's possible to extract it using `getMatchedCSSRules`, but it will be a lot of work.
